### PR TITLE
Refine for mlflow flavor support

### PR DESCRIPTION
**Pull request by brynn-code**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: src/promptflow/promptflow/_sdk/_serving/flow_invoker.py [hunk 1]
```diff
@@ -25,8 +26,8 @@ class FlowInvoker:
     """
     The invoker of a flow.
 
-    :param flow: The path of the flow.
-    :type flow: str
+    :param flow: The path of the flow, or the flow loaded by load_flow().
+    :type flow: [str, Flow]
```
Comment (0mza987) [inline src/promptflow/promptflow/_sdk/_serving/flow_invoker.py:30]: use `~promptflow.xxx.flow`?

File: src/promptflow/promptflow/_sdk/_serving/flow_invoker.py [hunk 2]
```diff
@@ -36,8 +37,7 @@ class FlowInvoker:
     def __init__(
         self, flow: str, connection_provider: [str, Callable] = None, streaming: Union[Callable[[], bool], bool] = False
     ):
-        self.flow_dir = flow
-        self.flow_entity = load_flow(self.flow_dir)
+        self.flow_entity = flow if isinstance(flow, Flow) else load_flow(source=flow)
```
Comment (0mza987) [inline src/promptflow/promptflow/_sdk/_serving/flow_invoker.py:40]: `flow` parameter annotation is only `str` in the interface.

Comment (brynn-code) [reply src/promptflow/promptflow/_sdk/_serving/flow_invoker.py:40]: Yes, I'm not sure if we need to expose it.

Comment (brynn-code) [reply src/promptflow/promptflow/_sdk/_serving/flow_invoker.py:43]: added

**Review by 0mza987 — DISMISSED** [review]

**Review by zhengfeiwang — APPROVED** [review]

**Review by YingChen1996 — APPROVED** [review]

**Review by wangchao1230 — APPROVED** [review]

**Pull request merged**